### ISSUE TITLE
Reduce scope of junit

### ISSUE THREAD

**Issue by sbrunecker** [mailjet/mailjet-apiv3-java#117]
Please consider reducing the scope of JUnit dependency to test, i.e.

```
<dependency>
    <groupId>junit</groupId>
    <artifactId>junit</artifactId>
    <version>4.11</version>
    <scope>test</scope>
</dependency>
```

**Comment by jmfsilva** [comment]
I've added a PR to fix this: #123

**Comment by goacoustic** [comment]
Hi guys, PR is merged, thanks!
